nuxeo-features/nuxeo-platform-forms-layout-export/src/test/java/org/nuxeo/ecm/platform/forms/layout/export/TestLayoutResource.java

```diff
@@ -0,0 +1,78 @@
+/*
+ * (C) Copyright 2018 Nuxeo (http://nuxeo.com/) and others.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * Contributors:
+ *     Kevin Leturc <kleturc@nuxeo.com>
+ */
+package org.nuxeo.ecm.platform.forms.layout.export;
+
+import static javax.ws.rs.core.MediaType.APPLICATION_JSON;
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertTrue;
+
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.nuxeo.ecm.core.test.annotations.Granularity;
+import org.nuxeo.ecm.core.test.annotations.RepositoryConfig;
+import org.nuxeo.ecm.webengine.test.WebEngineFeature;
+import org.nuxeo.jaxrs.test.CloseableClientResponse;
+import org.nuxeo.runtime.test.runner.Deploy;
+import org.nuxeo.runtime.test.runner.Features;
+import org.nuxeo.runtime.test.runner.FeaturesRunner;
+import org.nuxeo.runtime.test.runner.Jetty;
+
+import com.sun.jersey.api.client.Client;
+import com.sun.jersey.api.client.ClientResponse;
+import com.sun.jersey.api.client.WebResource;
+
+/**
+ * @since 10.1
+ */
+@RunWith(FeaturesRunner.class)
+@Features({ WebEngineFeature.class })
+@Jetty(port = 18090)
+@RepositoryConfig(cleanup = Granularity.METHOD)
+@Deploy({ "org.nuxeo.ecm.platform.forms.layout.core", "org.nuxeo.ecm.platform.forms.layout.export",
+        "org.nuxeo.ecm.platform.forms.layout.client:OSGI-INF/layouts-framework.xml",
+        "org.nuxeo.ecm.platform.forms.layout.export.tests:OSGI-INF/layouts-test-contrib.xml" })
+public class TestLayoutResource {
+
+    private static final int CLIENT_TIMEOUT = 5 * 60 * 1000; // 5 minutes
+
+    protected WebResource webResource;
+
+    @Before
+    public void initClient() {
+        Client client = Client.create();
```

Comment:
You should probably use `JerseyClienthHelper` or `HttpClientTestRule` here to use the underlying `ApacheHttpClient4Handler` instead of the default `URLConnectionClientHandler`.

Reply:
indeed, thanks 👍